framework/lib/test_generation/bin/_tool.source

```diff
@@ -22,56 +24,58 @@ die() {
 # Check whether an environment variable is set -- if not, die with an error message
 check_env() {
     local var="$1"
-    local val=$(eval echo \$$var)
+    local val
+    # shellcheck disable=SC2086 # would quoting even work?
+    val=$(eval echo \$$var)
```

Comment:
If the suggested quoting is

```
 val="$(eval echo \$$var)"
```

then that is desirable.  I don't know whether quoting is desirable/possible around `\$$var`.  This is weird enough code that disabling a warning seems OK.